lib/hanami/action/mime.rb

```diff
@@ -207,7 +211,14 @@ def format
       #     end
       #   end
       def content_type
```

Comment:
Great feedback @cyphactor, I'm going to make some modifications based on this today.